pkg/job/job.go

```diff
@@ -51,6 +51,7 @@ type Config struct {
 	RetryAttempts       int           `envconfig:"JOB_RETRY_ATTEMPTS" default:"30"`
 	ImageBuilder        string        `envconfig:"IMAGE_BUILDER" default:"quay.io/ocpmetal/assisted-iso-create:latest"`
 	Namespace           string        `envconfig:"NAMESPACE" default:"assisted-installer"`
+	S3SecretName        string        `envconfig:"S3_SECRET_NAME" default:"assisted-installer-s3"`
 	S3EndpointURL       string        `envconfig:"S3_ENDPOINT_URL" default:"http://10.35.59.36:30925"`
 	S3Bucket            string        `envconfig:"S3_BUCKET" default:"test"`
 	S3Region            string        `envconfig:"S3_REGION"`
```

Comment:
Can we delete those environment variables?

Reply:
No, they're still supplied to the assisted-service deployment with env vars:
https://github.com/openshift/assisted-service/blob/master/openshift/template.yaml#L142-L158

Reply:
Then why not to change that as well?

Reply:
They don't need to be changed. They're already linked as a secret reference rather than plain text.